### PR TITLE
US111035 - Outcomes picker cleanup

### PR DESCRIPTION
**Pull request by xiangqiyan**

### CODE REVIEW

File: d2l-asn-outcomes-picker.js [hunk 1]
```diff
@@ -111,7 +111,6 @@ class AsnOutcomesPicker extends LocalizedLitElement {
 			
 			asn-outcomes-picker-tree {
 				flex-grow: 1;
-				overflow-y: auto;
```
Comment (mpharoah-d2l) [inline d2l-asn-outcomes-picker.js]: Isn't this needed to have a scrollbar when the outcomes can't all fit?

Comment (xiangqiyan) [reply d2l-asn-outcomes-picker.js]: The body container has overflow-y set, so there will be a scrollbar.

Comment (xiangqiyan) [reply d2l-asn-outcomes-picker.js]: Although now you'd have to scroll all the way up to change the dropdown selection inside of having the dropdowns always on the screen. But i'm not sure if that's required

Comment (mpharoah-d2l) [reply d2l-asn-outcomes-picker.js]: It would be preferable to keep the scrollbar where it is now so that the dropdowns stay on screen

Comment (xiangqiyan) [reply d2l-asn-outcomes-picker.js]: Dave said to leave it as is so I'll revert the change.

**Pull request merged**